packages/decision-tree/src/components/Question/components/QuestionOption/QuestionOption.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { FC } from 'react';
+import { type QuestionOptionProps } from './types';
+import { Checkbox, Radio, Box, Text } from '@cmpsr/components';
+
+const componentsMap = {
+  checkbox: Checkbox,
+  radio: Radio,
+};
+
+export const QuestionOption: FC<QuestionOptionProps> = ({
+  componentType,
+  componentProps,
+  label,
+  subLabel,
+  ...props
+}) => {
+  return (
+    <Box
+      {...props}
+      border="1px solid #666"
+      borderColor="ui-element-outline-default"
+      background="background-action-default"
+      borderRadius="radii-4"
+      mb="spacer-4"
+      mx="auto"
+    >
+      {React.createElement(componentsMap[componentType], {
+        ...componentProps,
+        display: 'flex',
+        flexDirection: 'row-reverse',
```

Comment:
is `row-reverse` needed? From the accessibility point of view this should be avoided

Reply:
Yes, because we have no way to put the text on the left side with the current composer options, it is either this, or create a PR for composer to allow left sided Text